www/modules/centreon-open-tickets/views/rules/index.php

```diff
@@ -43,37 +43,34 @@
 try {
     switch ($o) {
         case 'a':
-            require_once 'form.php';
+            include_once 'form.php';
```

Comment:
you shouldn't replace require with include, as the files called are mandatory (form.php and list.php, ...) and the script can't work without them.